src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractReportServiceImpl.java

```diff
@@ -260,9 +261,11 @@ protected String getNullSafeReportIdFromItem(ConcurBatchReportLineValidationErro
         return StringUtils.defaultIfBlank(errorItem.getReportId(), BLANK_REPORT_ID_SUBSTITUTE);
     }
 
-    protected void writeSubTitleForErrorSubReport(String subTitle) {
+    protected void writeSubTitleForErrorSubReport(String[] subTitles) {
```

Comment:
Another area that could benefit from converting "subTitles" to a var-args parameter.